src/main/java/com/minecolonies/entity/ai/citizen/guard/EntityAIMeleeGuard.java

```diff
@@ -1,8 +1,11 @@
 package com.minecolonies.entity.ai.citizen.guard;
 
+import com.minecolonies.colony.buildings.AbstractBuilding;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.colony.buildings.AbstractBuilding'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)